doc/snippets/Microsoft.Data.SqlClient/SqlDataAdapter.xml

```diff
@@ -1,501 +1,1374 @@
+        <code language="c#">
+          using System;
+          using System.Data;
+          using Microsoft.Data.SqlClient;
+          
+          class Program
+          {
+              static void Main()
+              {
+              }
+            
+              public static SqlDataAdapter CreateSqlDataAdapter(string commandText, string connectionString)
+              {
+                  SqlDataAdapter adapter = new SqlDataAdapter(commandText, connectionString);
+                  SqlConnection connection = adapter.SelectCommand.Connection;
+          
+                  adapter.MissingSchemaAction = MissingSchemaAction.AddWithKey;
+          
+                  // Create the commands.
+                  adapter.InsertCommand = new SqlCommand(
+                      "INSERT INTO Customers (CustomerID, CompanyName) " +
+                      "VALUES (@CustomerID, @CompanyName)", connection);
+          
+                  adapter.UpdateCommand = new SqlCommand(
+                      "UPDATE Customers SET CustomerID = @CustomerID, CompanyName = @CompanyName " +
+                      "WHERE CustomerID = @oldCustomerID", connection);
+          
+                  adapter.DeleteCommand = new SqlCommand(
+                      "DELETE FROM Customers WHERE CustomerID = @CustomerID", connection);
+          
+                  // Create the parameters.
+                  adapter.InsertCommand.Parameters.Add(
+                      "@CustomerID",
+                      SqlDbType.Char,
+                      5,
+                      "CustomerID");
+                  adapter.InsertCommand.Parameters.Add(
+                      "@CompanyName",
+                      SqlDbType.VarChar,
+                      40,
+                      "CompanyName");
+          
+                  adapter.UpdateCommand.Parameters.Add(
+                      "@CustomerID",
+                      SqlDbType.Char,
+                      5,
+                      "CustomerID");
+                  adapter.UpdateCommand.Parameters.Add(
+                      "@CompanyName",
+                      SqlDbType.VarChar,
+                      40,
+                      "CompanyName");
+                  adapter.UpdateCommand.Parameters.Add(
+                      "@oldCustomerID",
+                      SqlDbType.Char,
+                      5,
+                      "CustomerID"
+                  ).SourceVersion = DataRowVersion.Original;
+          
+                  adapter.DeleteCommand.Parameters.Add(
+                      "@CustomerID",
+                      SqlDbType.Char,
+                      5,
+                      "CustomerID"
+                  ).SourceVersion = DataRowVersion.Original;
+          
+                  return adapter;
+              }
+          }
+        </code>
+      </example>
+    </ctorSelectCommandTextSelectConnectionString>
+    <AddToBatch>
+      <param name="command">
+        To be added.
```

Comment:
do we plan to keep these? I know others were removed

Reply:
So, I removed the remarks with "To be added" under the rationale that if we haven't added them now, we probably don't need them. But in those scenarios we already had other comment blocks, so there would still be comments in the output.

On the other hand, if I remove these "to be added" comments, the corresponding declaration no longer has any comments. That might cause the build to fail, but if it doesn't we don't have any easy way of searching for missing comments. So I'm leaving them here for now so we can easily go back through and add them at a later date (not as part of the work for #2013!)